docs/_layouts/ds4/default.html

```diff
@@ -9,6 +9,10 @@
         <link type="text/css" rel="stylesheet" href="static/ds4/docs.css"/>
     </head>
     <body>
+        <div class="ds6-banner" style="background-color: #f5f5f5; border-bottom: 2px solid #111820; font-weight: bold; position: fixed; z-index: 2; padding: 10px 16px; top: 0; width: 100vw;">
```

Comment:
Can we move the inline styles to a class?

Reply:
Sure. I just didn't want to spend a lot of time on the DS4 side of things, and to limit the number of files changed.

Reply:
And ... this page felt quick and dirty, and wasn't part of my original plan. Fixing.